src/Symfony/Component/Security/Http/LoginLink/LoginLinkHandlerInterface.php

```diff
@@ -25,7 +25,7 @@ interface LoginLinkHandlerInterface
     /**
      * Generate a link that can be used to authenticate as the given user.
      */
-    public function createLoginLink(UserInterface $user): LoginLinkDetails;
+    public function createLoginLink(UserInterface $user, ?string $locale = null): LoginLinkDetails;
```

Comment:
Same

Reply:
That's a BC break, so it's not possible to change the interface. See how we deal with such a case in some other occurrences in the code base.

Reply:
> That's a BC break, so it's not possible to change the interface. See how we deal with such a case in some other occurrences in the code base.

Maybe it's possible to add a setLocale method and a $locale property in LoginLinkHandler class and FirewallAwareLoginLinkHandler class. What do you think of that?